oemr-selenium/logout.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+from test_utils import *
+
+class TestWebsite_logout:
+    @pytest.fixture(autouse=True)
+    def browser_setup_and_teardown(self):
+        options = Options()
+        self.browser = webdriver.Chrome(options=options)
+        self.browser.maximize_window()
+        self.browser.implicitly_wait(10)
+
+        yield
+        self.browser.close()
+        self.browser.quit()
+
+    @pytest.mark.parametrize("url, username, password", read_configurations_from_file("secret.json"))
+    def test_logout(self, url, username, password):
+        self.browser.get(url)
+
+        self.browser.find_element(By.ID, 'authUser').send_keys(username)
+        self.browser.find_element(By.ID, "clearPass").send_keys(password)
+        self.browser.find_element(By.ID, "login-button").submit()
+
+        assert "https://in-info-web18.luddy.iupui.edu/interface/main/tabs/main.php" in self.browser.current_url
```

Comment:
Dont hardcode

Reply:
Modified the secret.json structure to include this value since it is the same for all the tests. Added a new utility function to read this from secret.json.